commons/zenoh-config/src/lib.rs

```diff
@@ -51,6 +52,39 @@ pub use mode_dependent::*;
 pub mod connection_retry;
 pub use connection_retry::*;
 
+/// The global unique id of a zenoh peer.
```

Comment:
Why not just use `#[doc(hidden)]` on `zenoh_protocol::core::ZenohId` internal methods, instead of writing a new type?

Reply:
`#[doc(hidden)]` methods are not visible in autocompletion either in RustRovers, or in VSCode.

Reply:
And technically these methods are still available to the user, why they should not be. They are even not in "internal" part of api - it's just some internal methods which leaked due to our quite arbitrary crate separation. So better to hide them correctly.

Reply:
Ok.